src/applications/survivors-benefits/config/chapters/06-financial-information/incomeAndAssets/grossMonthlyIncomePages.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+import {
+  titleUI,
+  radioUI,
+  textUI,
+  radioSchema,
+  currencyUI,
+  currencySchema,
+  arrayBuilderYesNoUI,
+  arrayBuilderYesNoSchema,
+} from 'platform/forms-system/src/js/web-component-patterns';
+import { VaTextInputField } from 'platform/forms-system/src/js/web-component-fields';
+
+const recipientTypeLabels = {
+  SURVIVING_SPOUSE: 'Surviving spouse',
+  VETERANS_CHILD: 'Veteran’s child',
+  CUSTODIAN: 'Custodian',
+  CUSTODIAN_SPOUSE: 'Custodian’s spouse',
+  OTHER: 'Other',
```

Comment:
Note in Figma says other is removed. Not sure how that affects the conditionals.